src/sentry/newsletter/remote.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
+
+import logging
+from uuid import uuid4
+from hashlib import sha1
+
+from sentry.http import safe_urlopen
+from sentry.newsletter.base import Newsletter
+from sentry.utils import json
+
+logger = logging.getLogger('sentry.newsletter')
+
+
+def get_install_id():
+    from sentry import options
+    install_id = options.get('sentry:install-id')
+    if not install_id:
+        install_id = sha1(uuid4().bytes).hexdigest()
+        options.set('sentry:install-id', install_id)
+    return install_id
+
+
+class RemoteNewsletter(Newsletter):
+    enabled = True
+    endpoint = 'https://sentry.io/remote/newslettersubscription/'
```

Comment:
tbh id probably just call it newsletter, but thats only because i think the name is ugly this way ;)